ghost/stack.yml

```
@@ -5,22 +5,32 @@ services:
   ghost:
     image: ghost:4-alpine
     restart: always
+    depends_on:
+      - ghost-db
     ports:
       - 8080:2368
     environment:
       # see https://ghost.org/docs/config/#configuration-options
       database__client: mysql
-      database__connection__host: db
+      database__connection__host: ghost-db
       database__connection__user: root
       database__connection__password: example
       database__connection__database: ghost
       # this url value is just an example, and is likely wrong for your environment!
       url: http://localhost:8080
       # contrary to the default mentioned in the linked documentation, this image defaults to NODE_ENV=production (so development mode needs to be explicitly specified if desired)
       #NODE_ENV: development
+    volumes:
+      - some-ghost-data:/var/lib/ghost/content
 
-  db:
+  ghost-db:
```

I don't think we really need to change this, do we?  It's already scoped to this stack (in both compose and swarm).

```suggestion
  db:
```